autogpt_platform/backend/test_disabled_block_bypass.py

```
@@ -0,0 +1,295 @@
+        "data": {"code": "# Should be rejected\nprint('hello')"},
+        "input": {"code": "# Should be rejected\nprint('hello')"},
+    }
+
+    resp = requests.post(
+        f"{base_url}/api/v1/blocks/{BLOCK_INSTALLATION_BLOCK_ID}/execute",
+        headers=headers,
+        json=block_payload,
+    )
+
+    if resp.status_code == 403:
+        print(f"  PASS - Server rejected direct execution (HTTP 403)")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)